openapi/multi-yaml/index.yaml

```diff
@@ -79,6 +79,223 @@ tags:
+      This feature enables platforms to maintain a single source of stored payment methods while processing payments through different sub-accounts.
+
+      ### Enabling Platform Wallet Accounts
+      Contact JustiFi support to enable the `platform_wallet_account` setting for your designated wallet account.
+      Once enabled, you can:
+      - Store payment methods in the designated wallet account
+      - Use these payment methods across your platform's sub-accounts
+      - Group payment methods for easier management. See [PaymentMethodGroups](http://localhost:3001/api-spec#tag/Payment-Method-Groups)
+
+      ### Using Platform Wallet Payment Methods
+      1. Create payment methods in your wallet account using the [Payment Methods API](http://localhost:3001/api-spec#tag/Payment-Methods/operation/CreatePaymentMethod):
+      ##### Example Request: Create Payment Method in Wallet Account
+      ```json
+      {
+        "payment_method": {
+          "card": {
+            "name": "Lindsay Whalen",
+            "number": "4242424242424242",
+            "verification": "123",
+            "month": 5,
+            "year": 2025,
+            "address_postal_code": "55555"
+          }
+        }
+      }
+      ```
+
+      ##### Example Response: Wallet Payment Method Created
+      ```json
+      {
+        "id": "pm_abc123xyz",
+        "type": "payment_method",
+        "data": {
+          "signature": "3aGWnUznQ",
+          "account_id": "acc_123wallet",
+          "status": "valid",
+          "card": {
+            "id": "pm_abc123xyz",
+            "name": "Lindsay Whalen",
+            "acct_last_four": "4242",
+            "brand": "visa",
+            "token": "pm_abc123xyz",
+            "month": "5",
+            "year": "2025",
+            "metadata": "info",
+            "address_line1_check": "pass",
+            "address_postal_code_check": "pass",
+            "bin_details": {
+              "type": "Credit",
+              "card_brand": "Visa",
+              "card_class": "Consumer",
+              "country": "United States of America",
+              "issuer": "WELLS FARGO BANK, N.A.",
+              "funding_source": "Credit"
+            }
+          }
+        }
+      }
+      ```
+      2. Process payments for any sub-account using the wallet payment method token. The system automatically:
+        - Validates the wallet payment method can be used by the sub-account
+        - Creates a cloned payment method owned by the sub-account
+        - Returns the new sub-account-specific token in the payment response
+
+        ### [Payments API](http://localhost:3001/api-spec#tag/Payments/operation/CreatePayment) Example
+
+        ##### Example Request: Create Payment Using Wallet Token
+        ```json
+        {
+          "amount": 1000,
+          "currency": "usd",
+          "payment_method": {
+            "token": "pm_abc123xyz"
+          }
+        }
+        ```
```

Comment:
Hm, I wonder if this should include the whole request, like we do [here for checkouts, under Create a checkout](https://docs.justifi.tech/api-spec#tag/Checkout-via-API)?
Or change the headline to `Example request body:...`
I think I'd prefer the whole request. This way it shows the `Sub-Account` header as well.